index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+            <a href="#">My Logo</a>
+        </div>
+        <nav>
+            <img src="assets/IconMenu.png" alt="">
+        </nav>
+    </header>
+    <section class="headline">
+        <h2>I'm Ashraf Glad to see you!</h2>
```

Comment:
- [ ] Kindly consider using `h1` for the main title of your website to improve SEO.